accessibility-checker.php

```diff
@@ -134,9 +133,8 @@
 /**
  * Filters and Actions
  */
-add_action( 'admin_enqueue_scripts', 'edac_admin_enqueue_scripts' );
-add_action( 'admin_enqueue_scripts', 'edac_admin_enqueue_styles' );
-add_action( 'wp_enqueue_scripts', 'edac_enqueue_scripts' );
+add_action( 'admin_enqueue_scripts', array( 'EDAC\Admin\Enqueue_Admin', 'enqueue' ) );
+add_action( 'wp_enqueue_scripts', array( 'EDAC\Inc\Enqueue_Frontend', 'enqueue' ) );
```

Comment:
Shouldn't this be moved to the `class-plugin.php` so it's never executed unless on the front-end, consistency, and to keep the main plugin file clean?

Reply:
@SteveJonesDev Good catch, thanks.